### PR TITLE
Properly encapsulate require_once for app.php

### PR DESCRIPTION
**Pull request by juliusknorr**
If an app has an appinfo/app.php file, that includes a variable called $app, this would lead to a type error when calling getAppInfo otherwise.

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]
I had exactly this just minutes ago :)

File: lib/private/legacy/app.php [hunk 1]
```diff
@@ -133,6 +133,7 @@ public static function loadApps($types = null) {
 	 * load a single app
 	 *
 	 * @param string $app
+	 * @throws Exception
```
Comment (MorrisJobke) [inline lib/private/legacy/app.php]: This should be added to `requireAppFile` 😉

Comment (juliushaertl) [reply lib/private/legacy/app.php]: Added an annotation for Error to `requireAppFile`, but Exception can be thrown by `loadApps` though `self::disable`

**Comment by juliushaertl** [comment]
stable12 https://github.com/nextcloud/server/pull/8632
stable13 https://github.com/nextcloud/server/pull/8631

**Pull request merged**